content/atomics_intro.tex

```diff
@@ -52,18 +49,26 @@
   \begin{center}
     \begin{tabular}{|l|l|}
       \hline
-      \TYPE & \TYPENAME\\
-      \hline
-      float & float\\
-      \hline
-      double & double\\
-      \hline
-      int & int\\
-      \hline
-      long & long\\
-      \hline
-      long long & longlong\\
-      \hline
+      \TYPE              & \TYPENAME  \\ \hline
+      float              & float      \\ \hline
+      double             & double     \\ \hline
+      int                & int        \\ \hline
+      long               & long       \\ \hline
+      long long          & longlong   \\ \hline
+%\newtext {
+      unsigned int       & uint       \\ \hline
+      unsigned long      & ulong      \\ \hline
+      unsigned long long & ulonglong  \\ \hline
+      int32\_t           & int32      \\ \hline
+      int64\_t           & int64      \\ \hline
+      uint32\_t          & uint32     \\ \hline
+      uint64\_t          & uint64     \\ \hline
+      size\_t            & size       \\ \hline
+      ptrdiff\_t         & ptrdiff    \\ \hline
+      wchar\_t           & wchar      \\ \hline
```

Comment:
Adding ```wchar_t``` to the AMO types makes me nervous because it can be smaller than ```int```, thereby introducing a new requirement to the AMO interface.  There should not be performance impact to the other AMO routines, because we isolate now by type.  Please correct me if I'm wrong, but adding the ```wchar_t``` type seems to be of little value to users and an unnecessary distraction to implementors.